docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,259 @@
+    
+[**NASA Earthdata**](https://search.earthdata.nasa.gov) provides among many others harmonized Landsat 8 and Sentinel-2 dataset. It requires [NASA EarthData registration](https://urs.earthdata.nasa.gov/users/new) and download is possible via Graphical web interface and bulk download.
+
+**Finnish Sentinel image mosaics** are provided by the [Finnish Environment Institute, SYKE](https://www.syke.fi/en-US) and the [Finnish Meteorological Institute, FMI](https://en.ilmatieteenlaitos.fi/) and partly available in Puhti: `/appl/data/geo/sentinel`. Among others, [Sentinel-2 image index mosaics](https://ckan.ymparisto.fi/dataset/sentinel-2-image-index-mosaics-s2ind-sentinel-2-kuvamosaiikit-s2ind)  and [Sentinel-1 SAR-image mosaics](https://ckan.ymparisto.fi/dataset/sentinel-1-sar-image-mosaic-s1sar-sentinel-1-sar-kuvamosaiikki-s1sar) are available.
+
+[**Google Cloud**](https://cloud.google.com) provides some [public EO datasets in Google Cloud Storage](https://cloud.google.com/storage/docs/public-datasets), inlcuding Sentinel-2 L1C and Landsat Collection 1 data. Google Cloud also offers the possibility for virtual machines, with both CPU and GPU.
+ and <a href="https://cloud.google.com/storage/docs/public-datasets/landsat" aria-label="Landsat: Collection 1 data on Google cloud storage">Landsat: Collection 1</a>. Data can be downloaded from here for example with [FORCE](../../../apps/force.md).
+
+[**Terramonitor**](https://www.terramonitor.com/services/analysis-ready) provides pre-prosessed, analysis ready Sentinel-2 data from Finland available between 2018-2020.[Contact Terramonitor about pricing](https://www.terramonitor.com/services/analysis-ready#contact)
+
+[**Sentinelhub**](https://www.sentinel-hub.com/explore/) provides among others worldwide <a href="https://collections.sentinel-hub.com/sentinel-2-l2a/" aria-label="Sentinel 2 on sentinelhub">Sentinel 2</a> and <a href="https://collections.sentinel-hub.com/sentinel-1-grd/" aria-label="Sentinel 1 on sentinelhub">Sentinel 1</a> data. Requires [sentinelhub subscription](https://www.sentinel-hub.com/pricing/).
+
+!!! default "STAC"
+
+    Many data providers and companies also provide a Spatio Temporal Asset Catalog (STAC) of their and other datasets. These catalogues help in finding available data based on time and location with the possibility for multiple additional filters, such as cloud cover and resolution. The [STAC Index](https://www.stacindex.org/) provides a nice overview of available catalogues from all over the world. The STAC Index page also includes many resources for learning and utilizing STAC. Check out also CSC's [examples for utilizing STAC from Python](https://github.com/csc-training/geocomputing/blob/master/python/STAC).
+
+## Where can I store the data?
```

Comment:
I wonder if it would be better for example here put a heading a la "Working with EO data at CSC" The rest at practical level about CSC other than the what to consider..